src/forces.f90

```diff
@@ -4,531 +4,850 @@
-
-  integer :: nvol,iforces
-  real(mytype),save,allocatable,dimension(:,:,:) :: ux01, uy01, ux11, uy11, ppi1
-  real(mytype),allocatable,dimension(:) :: xld,xrd,yld,yud
-  integer,allocatable,dimension(:) :: icvlf,icvrt,jcvlw,jcvup
-  integer,allocatable,dimension(:) :: icvlf_lx,icvrt_lx,icvlf_ly,icvrt_ly
-  integer,allocatable,dimension(:) :: jcvlw_lx,jcvup_lx,jcvlw_ly,jcvup_ly
-
-  character(len=*), parameter :: io_restart_forces = "restart-forces-io", &
-       resfile = "restart-forces"
-  
-contains
-
-  subroutine init_forces
-
-    USE decomp_2d_io, only : decomp_2d_register_variable, decomp_2d_init_io
-    USE param
-    USE variables
-    implicit none
-
-    integer :: iv,stp1,stp2,h
-
-    call alloc_x(ux01)
-    call alloc_x(uy01)
-    call alloc_x(ux11)
-    call alloc_x(uy11)
-    call alloc_x(ppi1)
-
-    ux01 = zero
-    uy01 = zero
-    ux11 = zero
-    uy11 = zero
-
-    allocate(icvlf(nvol),icvrt(nvol),jcvlw(nvol),jcvup(nvol))
-    allocate(icvlf_lx(nvol),icvrt_lx(nvol),icvlf_ly(nvol),icvrt_ly(nvol))
-    allocate(jcvlw_lx(nvol),jcvup_lx(nvol),jcvlw_ly(nvol),jcvup_ly(nvol))
-
-    !     Definition of the Control Volume
-    !*****************************************************************
-    !! xld,xrd,yld,yud: limits of control volume (!!don't use cex and cey anymore!!)
-
-    do iv=1,nvol
-       ! ok for istret=0 (!!to do for istret=1!!)
-       icvlf(iv) = nint(xld(iv)/dx)+1
-       icvrt(iv) = nint(xrd(iv)/dx)+1
-       if (istret.eq.0) then 
-         jcvlw(iv) = nint(yld(iv)/dy)+1
-         jcvup(iv) = nint(yud(iv)/dy)+1
-       else
-         stp1=0
-         stp2=0
-         do h = 1, ny-1  
-           if ((-yp(h+1)-yp(h)+two*yld(iv)).lt.(yld(iv)-yp(h)).and.(stp1.eq.0)) then 
-             jcvlw(iv) = h+1
-             stp1=1
-           endif
-           if ((-yp(h+1)-yp(h)+two*yud(iv)).lt.(yud(iv)-yp(h)).and.(stp2.eq.0)) then
-             jcvup(iv) = h
-             stp2=1 
-           endif
-         enddo
-       endif
-       icvlf_lx(iv) = icvlf(iv)
-       icvrt_lx(iv) = icvrt(iv)
-       jcvlw_lx(iv) = max(jcvlw(iv)+1-xstart(2),1)
-       jcvup_lx(iv) = min(jcvup(iv)+1-xstart(2),xsize(2))
-       icvlf_ly(iv) = max(icvlf(iv)+1-ystart(1),1)
-       icvrt_ly(iv) = min(icvrt(iv)+1-ystart(1),ysize(1))
-       jcvlw_ly(iv) = jcvlw(iv)
-       jcvup_ly(iv) = jcvup(iv)
-    enddo
-
-    if (nrank==0) then
-       write(*,*) '========================Forces============================='
-       write(*,*) '                       (icvlf)      (icvrt) '
-       write(*,*) '                (jcvup) B____________C '
-       write(*,*) '                        \            \ '
-       write(*,*) '                        \     __     \ '
-       write(*,*) '                        \    \__\    \ '
-       write(*,*) '                        \            \ '
-       write(*,*) '                        \       CV   \ '
-       write(*,*) '                (jcvlw) A____________D '
+   use decomp_2d_constants
```

Comment:
Is it really needed to modify over 4500 lines of code for this PR ? Please stick to the existing format and keep the modification as small as possible.

Reply:
I have added about 500 lines of code in `force.f90` which has the maximum change. I don't think 4500 lines have been modified. Also, I have kept it as small as possible.

Reply:
[src/forces.f90](https://github.com/xcompact3d/Incompact3d/pull/260/files#diff-d8b154226eaf2ab5cccc3e00c96d3584673f7d874c191643f31c00399e986739)
Viewed
@@ -4,531 +4,850 @@

4531 lines are removed and 4850 lines are added.

Reply:
I will check why the number of lines is so high and let you know. Thanks for the review !

Reply:
I do not know the exact number of lines modified. However, it looks like you have changed the number of spaces or the formatting of the file. The amount of modification is reduced when the box "Hide whitespace" is ticked.

It is not a good practice to combine code modification and format modification. Please open a new PR and keep the existing format as much as possible